### ISSUE TITLE
Identical errors reported in "PSR2.Methods.FunctionCallSignature" sniff

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#422]
Code:
```php
$supply_order_item =& $this->Application->recallObject(
	'supply-order-item.receiving', null, array('skip_autoload' => true)
);
```
I'm receiving 2 `Only one argument is allowed per line in a multi-line function call` (error code: `MultipleArguments`) errors on same line (where parameters are). I guess it's for 2nd and 3rd parameters, that should each be on it's own line.

Having same errors on one line doesn't really help much. I think, that we should:

* either show parameter (number or it's contents) which error belongs to
* or don't report more than 1 such error on same line

**Comment by gsherwood** [comment]
The one thing you can't see on the screen reports is that PHPCS also records the column at which the error occurred. So if you are integrating PHPCS with an IDE using something like the CSV, XML or JSON reports, you can highlight the token that caused the error. So in this case, you can highlight each param and show an error message for it.

For this reason, I want to keep this sniff the way it is.

**Comment by aik099** [comment]
Then it's a bug in PhpStorm, that highlights whole line as an error. I saw, in the PHP_CodeSniffer code, that token (having line and column number) is sent to the error, but never saw the actual column displayed anywhere.

**Comment by aik099** [comment]
I've reported issue to PhpStorm: https://youtrack.jetbrains.com/issue/WI-26083
